apps/condo/domains/user/components/auth/RegisterForm.tsx

```diff
@@ -43,6 +43,9 @@ export const RegisterForm: React.FC<IRegisterFormProps> = ({ onFinish }) => {
     const EmailIsAlreadyRegisteredMsg = intl.formatMessage({ id: 'pages.auth.EmailIsAlreadyRegistered' })
     const ConfirmActionExpiredError = intl.formatMessage({ id: 'pages.auth.register.ConfirmActionExpiredError' })
 
+    const PASSWORD_MSG_LABEL = <label style={{ whiteSpace:'break-spaces' }}>{PasswordMsg}</label>
```

Comment:
Why it’s looks like const? Do we really use this style ?

Reply:
If u're talking about making it a constant, this is for readability by the advice of Pavel.
If u're talking about creating a label, this is because the styles of the label embedded in the form cannot be changed